dlt/extract/extract.py

```diff
@@ -75,6 +75,7 @@ def choose_schema() -> Schema:
         """Except of explicitly passed schema, use a clone that will get discarded if extraction fails"""
         if schema:
             schema_ = schema
+        # if we have a default schema name, use it but do not collect any info from the existing schema
```

Comment:
I think is is a wrong comment? we still clone pipeline schema

Reply:
ah yes, i had to revert this but forgot to remove this comment